libs/blocks/merch/merch.js

```diff
@@ -1,57 +1,159 @@
-import { loadScript, getConfig, createTag } from '../../utils/utils.js';
+import {
+  loadScript,
+  getConfig,
+  createTag,
+} from '../../utils/utils.js';
 
-const { miloLibs, codeRoot, env } = getConfig();
-const base = miloLibs || codeRoot;
+const version = '1.12.0';
+const wcs = { apiKey: 'wcms-commerce-ims-ro-user-milo' };
+const envProd = 'prod';
+const ctaPrefix = /^CTA +/;
```

Comment:
one more nit: const values should be UPPERCASED_NAMES

Reply:
same for `SUPPORTED_LANGS` and `GEO_MAPPINGS` and those should be above `initialized`